src/main/java/data_gateway/EventEntityManager.java

```diff
@@ -103,4 +104,33 @@ public void markEventAsCompleted(long eventId) {
         }
         return eventReaderList;
     }
+
+
+    @Override
+    public void updateName(long id, String newName) {
+        for(Event event : eventList){
```

Comment:
Might be a good idea to add a private `getById` method to prevent this repetitive for loop (same for the TaskManager)